### ISSUE TITLE
Documentation of 5 Variations

### ISSUE THREAD

**Issue by rachael-tomaszewski** [orzubalsky/creative-computing-2018#98]
I'm a bit confused about the documenting of the 5 variations of our code, are we meant to screen record them individually and submit them?

**Comment by ellenyxr** [comment]
Yes I think we should do screen record since they are animations

**Comment by misteakes** [comment]
You can record them individually, but I remember Or saying we could have all 5 variations in one recording so we don't have to submit so many things. Up to you!

**Comment by chanrxia** [comment]
yup, 1 screenrecording of 5 variations of the shape, but do you guys know if wehave to show all 5 shapes changing or just 1?
